black.py

```diff
@@ -1160,9 +1160,11 @@ class Line:
     """Holds leaves and comments. Can be printed with `str(line)`."""
 
     depth: int = 0
-    leaves: List[Leaf] = Factory(list)
-    comments: Dict[LeafID, List[Leaf]] = Factory(dict)  # keys ordered like `leaves`
-    bracket_tracker: BracketTracker = Factory(BracketTracker)
+    leaves: List[Leaf] = field(default_factory=list)
+    comments: Dict[LeafID, List[Leaf]] = field(
+        default_factory=dict
+    )  # keys ordered like `leaves`
```

Comment:
Can you put this comment on the previous line by itself instead? (I'd personally prefer if Black did that automatically in this case.)

Reply:
Wow you'd think I'd have run the tests, huh?